src/openapi/common-responses.ts

```diff
@@ -0,0 +1,74 @@
+import { OpenAPIV3 } from 'openapi-types';
+
+export const NOT_READY_MSG =
+    'The Unleash Proxy has not connected to the Unleash API and is not ready to accept requests yet.';
+
+export const notReadyResponse: OpenAPIV3.ResponseObject = {
+    description: "The proxy isn't ready to accept requests yet.",
+    content: {
+        'text/plain': {
+            schema: {
+                type: 'string',
+                example: NOT_READY_MSG,
+            },
+        },
+    },
+} as const;
+
+export const unauthorizedResponse: OpenAPIV3.ResponseObject = {
+    description: 'Authorization information is missing or invalid.',
+} as const;
+
+export const emptySuccessResponse: OpenAPIV3.ResponseObject = {
```

Comment:
Could extract these to separate files, `empty-success-response.ts` etc, but pretty subjective. :smile:

Reply:
Hmm, could. Do we generally have a pattern here? On a subjective level, I think it makes sense to keep it as one of a group of common responses, but I don't necessarily disagree with the other way 💁🏼